firefox-ios/Client/TabManagement/TabWebViewPreview.swift

```diff
@@ -0,0 +1,113 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+
+final class TabWebViewPreview: UIView, ThemeApplicable {
+    // MARK: - UX Constants
+    private struct UX {
+        static let addressBarCornerRadius = CGFloat(8)
```

Comment:
Can we change the format here as this is how we are doing it in most if not all places
```suggestion
        static let addressBarCornerRadius: CGFloat = 8
```